src/app/not-found.tsx

```diff
@@ -31,7 +31,16 @@ export default function NotFoundPage() {
   }/issues/new?title=${encodeURIComponent(title)}&labels=${labels}`
 
   return (
-    <MainLayout>
+    <>
+      {/*
+        No support for metadata in not-found.tsx yet
+        https://github.com/vercel/next.js/pull/47328#issuecomment-1488891093
+      */}
+      <title>
+        Not Found | Please click the button to file a broken-link issue if this
```

Comment:
Let's make it a bit shorter. It's landing in a browser tab, right?

```suggestion
        GraphQL | Page Not Found
```

<img width="293" height="44" alt="Image" src="https://github.com/user-attachments/assets/73ff4408-378b-46c9-99bb-cc5e38622ed7" />